src/NovaDependencyContainer.php

```diff
@@ -153,6 +153,23 @@ public function resolveForDisplay($resource, $attribute = null)
                 continue;
             }
 
+            if (is_object($resource->{$dependency['property']}) && is_array($dependency['value'])) {
+                if (array_key_exists('value', $dependency) && in_array($resource->{$dependency['property']}->id, $dependency['value'])) {
```

Comment:
Similar here, you are already assuming `$dependency` has value as an array item, so is the check on L156 broken or are we safe to assume it will always be there?